plugins/tedge_apt_plugin/src/error.rs

```diff
@@ -9,18 +9,26 @@ pub enum InternalError {
     #[error(transparent)]
     FromUtf8(#[from] std::string::FromUtf8Error),
 
-    #[error("Parsing Debian package failed for `{file}`")]
-    ParsingError { file: String },
-
     #[error(transparent)]
     FromCsv(#[from] csv::Error),
 
+    #[error("Parsing Debian package failed for `{file}`, Error: {error}")]
+    ParsingError { file: String, error: String },
+
     #[error("Validation of {package} failed with version mismatch. Installed version: {installed}, Expected version: {expected}")]
     VersionMismatch {
```

Comment:
This error case is never used and is actually replaced by the more generic `MetaDataMismatch` error case.

=> It must be removed.

Reply:
Actually, it was getting used in the validate_version function but is now replaced with the `MetaDataMismatch` error.
If it's not used then `cargo clippy` in CI will catch.